homeassistant/components/notify/pushbullet.py

```diff
@@ -10,8 +10,8 @@
 import voluptuous as vol
 
 from homeassistant.components.notify import (
-    ATTR_DATA, ATTR_TARGET, ATTR_TITLE, ATTR_TITLE_DEFAULT,
-    PLATFORM_SCHEMA, BaseNotificationService)
+    ATTR_DATA, ATTR_TITLE, ATTR_TARGET, PLATFORM_SCHEMA, ATTR_TITLE_DEFAULT,
```

Comment:
Please don't change to non alphabetic order.

Reply:
Weird, that's `isort` with the default setting beside `multi_line_output=4`. It looks like `length_sort` is applied as well.